spec/mailers/previews/did_you_know_alert_mailer_preview.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class DidYouKnowAlertMailerPreview < ActionMailer::Preview
+  def message_to_instructors
+    dk_nomination_alert = Alert.where(type: 'DYKNominationAlert')
```

Comment:
You can do `Alert.find_by` here instead, to fetch the first record that matches.